apps/frontend/src/components/global/checklist/ChecklistRuleEdit.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <v-card class="mt-3 pt-4" height="42vh">
+    <v-card-text>
+      <v-row>
+        <v-col>
+          <v-select
+            v-model="selectedRule.status"
+            dense
+            label="Status"
+            :items="statusItems"
+            item-text="name"
+            item-value="value"
+            :item-color="statusColor(selectedRule.status)"
+          />
+        </v-col>
+        <v-col>
+          <v-select
+            v-model="selectedRule.severityOverride"
+            dense
+            label="Severity Override"
+            item-text="name"
+            item-value="value"
+            :items="checkPossibleOverrides(selectedRule.severity)"
+            @change="promptSeverityJustification"
+          />
+        </v-col>
+      </v-row>
+      <v-row class="mt-n8">
+        <v-col>
+          <strong>Finding Details: </strong><br />
+          <v-textarea
+            v-model="selectedRule.findingDetails"
+            solo
+            outlined
+            dense
+            no-resize
+            height="12vh"
+          />
+        </v-col>
+      </v-row>
+      <v-row class="mt-n10">
+        <v-col>
+          <strong>Comments: </strong>
+          <v-textarea
+            v-model="selectedRule.comments"
+            solo
+            outlined
+            dense
+            no-resize
+            height="8vh"
+          />
+        </v-col>
+      </v-row>
+    </v-card-text>
+  </v-card>
+</template>
+
+<script lang="ts">
+import {ChecklistVuln} from '@mitre/hdf-converters';
+import {Component, Prop, Vue} from 'vue-property-decorator';
+
+@Component
+export default class ChecklistRuleInfoBody extends Vue {
+  @Prop({type: Object, required: true}) selectedRule!: ChecklistVuln;
+  @Prop({type: Boolean, required: true}) sheet!: Boolean;
```

Comment:
i would appreciate it if you could make an issue to make this sheet thing a globally accessible component.  the usage would be to handle this justification stuff but also like passwords or anything else that requires *more*.  see the failure of ux that happens with the user info page / groups page esp in relation to api keys.

Reply:
Created: #4517